templates/activity.html

```diff
@@ -1,41 +1,52 @@
-<!doctype html>
-<html>
+{% extends 'includes.html' %}
+{% block content %}
+<div class="section no-pad-bot" id="index-banner">
+        <div class="container">
+            <h1 class="header center  col-test">Welcome</h1>
+            <div class="row center">
+                <h5 class="header col s12 light">Hello, world. You are at the {{ org_name }} community GCI website.</h5>
+            </div>
+        </div>
 
-<head>
-    <title>Community Activity</title>
-    <style>
-    canvas {
-        -moz-user-select: none;
-        -webkit-user-select: none;
-        -ms-user-select: none;
-    }
-    </style>
-    <meta http-equiv="content-type" content="text/html; charset=utf-8" />
```

Comment:
You'd want to keep this(Firefox and Lighthouse gives a warning otherwise)

Reply:
Done